### DISCUSSION TITLE
What should this extension be named for marketing purposes?

### DISCUSSION THREAD

**Discussion by aivct** [benf2004/Bing-Chat-History#1] [General]
Okay, so first of all the github name doesn't necessarily need to be the same as the marketing name,
but like, I'm just trying to think, what are users typing in when they are searching in the chrome/edge extension stores? 

![image](https://user-images.githubusercontent.com/79041835/218253476-90241433-c2a9-43f2-a146-91a2ac59a2bf.png)

ChatGPT has become a brand name at this point (god forbid it goes the way of Google, or what Nintendo almost did as a metonym for consoles)

To be clear, I still think they should be separate extensions, but do we want to sorta jump on the marketing hype and call it Bing ChatGPT? Or does it make more sense to clearly differentiate ourselves with Bing Chat History, but risk losing those clicks? Or maybe call it Bing Chat Plus?

Microsoft is so bad at marketing that users don't even know what to call their potentially new big product! Argh. What's it even called? @benf2004 What are the names you've seen that people are referring to Bing organically in the wild?

but idk... I know this whole business of naming is probably bike-shedding and we should probably focus on developing a useful extension... maybe we could later ask users to choose a name, lol.

**Comment by benf2004** [comment]
@aivct

["Vergecast: We tried Bing powered by ChatGPT AI"](https://www.theverge.com/2023/2/10/23594256/vergecast-we-tried-bing-powered-by-chatgpt-ai-and-things-got-dark)

["New Bing" & "new Bing"](https://www.bing.com/new)

["GPT-powered Bing chatbot"](https://www.businessinsider.com/gpt-ai-powered-bing-chatbot-secret-alias-codename-rules-2023-2)

"Bing" "chat mode of Microsoft Bing search" 
![screely-1676136329483](https://user-images.githubusercontent.com/12115686/218272099-61cdfb6f-57c9-4426-9c57-addd29d2a6ca.png)

Yeesh, it's a mess. I think in the title I will just do "Bing Chat History", but I'll include ChatGPT in the description. What do you think?

P.S. Don't try to call it by its codename "Sydney" or it gets sassy:
![screely-1676136522836](https://user-images.githubusercontent.com/12115686/218272265-7e82e2fa-25d1-490d-b6c9-c2c5629cce5e.png)

WHYY THE 😊 EMOJI!!! It's so passive aggressive hahaha.

**Reply by serniebanders** [reply]
Right? It's kinda concerning but I also love how it actually talks like a person. It's annoying when it's assertive about 100% incorrect things though.

**Reply by aivct** [reply]
It looks like search is not only based on the title, so adding ChatGPT in the description (ie, literally just Bing powered by ChatGPT or something) is probably a nice compromise.

Also you should totally post that image on reddit.

And I am *still* on the waitlist 😞
